domain/src/test/kotlin/no/nav/su/se/bakover/domain/grunnlag/FradragsgrunnlagTest.kt

```diff
@@ -74,38 +48,28 @@ internal class FradragsgrunnlagTest {
                 tilhører = FradragTilhører.BRUKER,
             ),
             opprettet = fixedTidspunkt,
-        ).valider(
-            behandlingsperiode,
-            false,
-        ) shouldBe Grunnlag.Fradragsgrunnlag.Validator.UgyldigFradragsgrunnlag.UgyldigFradragstypeForGrunnlag.left()
+        ) shouldBe Grunnlag.Fradragsgrunnlag.UgyldigFradragsgrunnlag.UgyldigFradragstypeForGrunnlag.left()
     }
 
     @Test
-    fun `ugyldig hvis et element i en liste er ugyldig`() {
-        listOf(
-            Grunnlag.Fradragsgrunnlag(
+    fun `kan lage gyldige fradragsgrunnlag`() {
+        Fradragstype.values().filterNot {
```

Comment:
Denne lista kunne kanskje eksistert i Fradragsgrunnlaget for behagelighetens skyld og kanskje for lesbarheten